.github/workflows/java.ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Java CI
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Do we even need this step? Bazel has the java jdk bundled, perhaps we could use that, but there would be several changes needed in other places needed, not sure if it's worth it.